package/Cargo.toml

```diff
@@ -21,7 +21,7 @@ doctest = false
 default = ["std", "serde"]
 arbitrary = ["dep:proptest-derive", "dep:proptest", "miden-assembly-syntax/arbitrary"]
 std = ["miden-assembly-syntax/std", "miden-core/std", "serde/std", "thiserror/std"]
-serde = ["dep:serde", "dep:serde-untagged", "miden-assembly-syntax/serde", "miden-core/serde"]
+serde = ["dep:serde", "dep:serde-untagged", "dep:serde_json", "miden-assembly-syntax/serde", "miden-core/serde"]
```

Comment:
I don't think we should be adding `serde_json` here except via `dev-dependencies` - AFAIK, it is only used for testing, and we don't want to impose it as an actual dependency of the published crate.

Reply:
It looks like `serde_json` is still being added as a regular dependency, rather than a dev-dependency.